### DISCUSSION TITLE
Should breadcrumbs be placed before the main content region?

### DISCUSSION THREAD

**Discussion by mbgower** [carbon-design-system/carbon#11090] [Accessibility]
In the [placement](https://v11.carbondesignsystem.com/components/breadcrumb/usage/#placement) section of the Usage tab for Breadcrumb, the following information appears:
> Breadcrumbs are placed in the top left portion of the page. They sit underneath the header and navigation, but above the page title.
![image](https://user-images.githubusercontent.com/14298245/160473457-52561d83-cf20-47e5-9001-05f0b1f05c30.png)

Despite this wording, there is no recommendation or information on where breadcrumbs are positioned relative to structural page regions.  I think Carbon should decide on a default of where the breadcrumbs are placed structurally.

Given the above description, I think it's arguable that they should precede the main content section.  Although the [ARIA Authoring Practices](https://w3c.github.io/aria-practices/#breadcrumb) is very non-specific in regards to this, the wording suggests this is an appropriate location.
> Breadcrumbs are often placed horizontally before a page's main content.

Either of the following is fine, but Carbon should decide on its preference.
1. Header
   1. breadcrumb navigation
1. Main

or
1. Header
1. Main
   1. breadcrumb navigation

It could even be that Carbon decides to bring them into the Shell UI section. A consideration may be where they fall visually when there is a left navigation section. There may be an opportunity for considering visual changes to Carbon to reinforce the decision.

**Comment by laurenmrice** [comment]
@mbgower 
- Which way is the most accessible in your opinion? I don't have a strong opinion between the two different options, but the current page you are on can be listed as a breadcrumb which then seems like breadcrumbs could live in the same region as the page you are on because they are referencing eachother. Our devs may also have an opinion on this.
- Should information regarding "regions" for breadcrumb placement be specified in the [Accessibility tab](https://carbondesignsystem.com/components/breadcrumb/accessibility)?

**Comment by mbgower** [comment]
@laurenmrice 
> Which way is the most accessible in your opinion?

I think either is fine. I lean slightly towards saying it is part of the header, simply because I think when I 'skip to main' content, i don't want to have to churn through the breadcrumb. Definitely worth a discussion with devs.
